appcelerator/titanium_mobile
# fix(android): set ListItem properties into proxy properties

- Set `ListItem` properties on components
- Move `composeContentDescription()` in appropriate place

##### TEST CASE
- Enable _Text-to-Speech_
- Select each item to confirm `accessibilityHint` has been set
```JS
const win = Ti.UI.createWindow({
    backgroundColor: 'white'
});
const listView = Ti.UI.createListView({
    templates: {
        template: {
            childTemplates: [{
                    type: 'Ti.UI.Label',
                    bindId: 'info',
                    properties: {
                        color: 'black',
                        font: {
                            fontFamily: 'Arial',
                            fontSize: '20dp',
                            fontWeight: 'bold'
                        },
                        left: '60dp',
                        top: 0
                    }
                },
                {
                    type: 'Ti.UI.Label',
                    bindId: 'es_info',
                    properties: {
                        color: 'gray',
                        font: {
                            fontFamily: 'Arial',
                            fontSize: '14dp'
                        },
                        left: '60dp',
                        top: '25dp'
                    }
                }
            ]
        }
    },
    defaultItemTemplate: 'template'
});

listView.setSections([
    Ti.UI.createListSection({
        headerTitle: 'Fruits / Frutas',
        items: [{
                info: {
                    text: 'Apple',
                    accessibilityLabel: 'Apple Label'
                },
                es_info: {
                    text: 'Manzana',
                    accessibilityHint: 'Manzana Hint'
                }
            },
            {
                info: {
                    text: 'Banana',
                    accessibilityLabel: 'Banana Label',
                    accessibilityValue: 'Banana Value'
                },
                es_info: {
                    text: 'Banana',
                    accessibilityHint: 'Banana Hint',
                    accessibilityValue: 'Banana Value'
                }
            }
        ]
    })
]);

win.add(listView);
win.open();
```

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-24415)
